app/models/enterprise.rb

```diff
@@ -80,7 +80,7 @@ class Enterprise < ActiveRecord::Base
   before_validation :set_unused_address_fields
   after_validation :geocode_address
 
-  before_save :check_instagram_pattern
+  validates :instagram, format: /\A@[a-zA-Z0-9._]{1,30}\z/, :allow_blank => true
```

Comment:
We try to keep the code style consistent. Can you change the last bit to `allow_blank: true`?